### PR TITLE
Add sequence diagrams for create event, create team

### PR DESCRIPTION
**Pull request by iryn34**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=777)

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [ ] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by VitalyiKhlivnyi — DISMISSED** [review]

File: documentation/sequence diagrams/event/Create Event sequence.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+participant UI CreateEvent Dialog
+participant EventView
+participant HttpRequest
+participant Event
+
+User->+UI EventList: Click button Create
+UI EventList->+UI CreateEvent Dialog: open CreateEvent Dialog
+User->UI CreateEvent Dialog: Enter data for creating a new event
+User->UI CreateEvent Dialog: Click button Submit
+alt not success
```
Comment (ashatna) [inline documentation/sequence diagrams/event/Create Event sequence.txt]: clarify, why success or not success in all document

File: documentation/sequence diagrams/event/Create Event sequence.txt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+UI CreateEvent Dialog->+EventView: post(request: HttpRequest, team_id: int)
```
Comment (ashatna) [inline documentation/sequence diagrams/event/Create Event sequence.txt]: I think, post request goes before success or not success statement

File: documentation/sequence diagrams/team/Create Team sequence.txt [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+participant TeamView
+participant HttpRequest
+participant Team
+
+User->+UI TeamList: Click button Create
+UI TeamList->+UI CreateTeam Dialog: open CreateTeam Dialog
+User->UI CreateTeam Dialog: Enter data for creating a new team
+User->UI CreateTeam Dialog: Click button Submit
+alt not success
+UI CreateTeam Dialog->+TeamView: post(request: HttpRequest)
```
Comment (ashatna) [inline documentation/sequence diagrams/team/Create Team sequence.txt]: post request before if/else statement
clarify success and not success statement

File: documentation/sequence diagrams/event/Create Event sequence.txt (hunk 2 above)
Comment (iryn34) [reply documentation/sequence diagrams/event/Create Event sequence.txt]: Done!

**Review by VitalyiKhlivnyi — DISMISSED** [review]

**Review by VitalyiKhlivnyi — APPROVED** [review]

**Pull request merged**